locales/en-US/app.ftl

```diff
@@ -498,8 +498,8 @@ StackSettings--implementation-all-stacks = All stacks
 StackSettings--implementation-javascript = JavaScript
 StackSettings--implementation-native = Native
 
-StackSettings--summarize = Summarize:
-StackSettings--call-tree-strategy-timing = Timing Data
+StackSettings--display-base-data = Use recorded data:
+StackSettings--call-tree-strategy-timing = Timings
```

Comment:
We can still change them like this because we haven't started translating the strings.

Reply:
Thought (somewhat of a suggestion): What about "Data source:" instead of "Use recorded data:".

I do like the change of Timing Data to just Timings

Reply:
Yeah, I wasn't super happy about "Use recorded data" anyway.